### ISSUE TITLE
[Feature Request] iOS Server

### ISSUE THREAD

**Issue by CSab6482** [BlueBubblesApp/BlueBubbles-Server#188]
I understand if this has to be placed in the backlog of the backlog of the backlog, but this is a feature that would help a lot of people who don't want to shell out the money for a Mac or who are worried about having a computer charger or PSU plugged in 24/7. Below are some comments from HowToMen's video.
![image](https://user-images.githubusercontent.com/53708281/119215756-aec96c80-ba9d-11eb-9a3a-64fb95e3155d.png)
![image](https://user-images.githubusercontent.com/53708281/119215770-c4d72d00-ba9d-11eb-8912-c596306e6325.png)
![image](https://user-images.githubusercontent.com/53708281/119215789-e59f8280-ba9d-11eb-8cca-4dc4abc9d36e.png)
![image](https://user-images.githubusercontent.com/53708281/119215798-f5b76200-ba9d-11eb-83c8-eb27fd08aeda.png)
![image](https://user-images.githubusercontent.com/53708281/119215802-010a8d80-ba9e-11eb-8d21-abc3d6d51dea.png)
![image](https://user-images.githubusercontent.com/53708281/119215821-12539a00-ba9e-11eb-8474-beaf58e59f65.png)
Also, this would help for use cases where something works differently on iMessage for macOS vs. iOS (see [here](https://github.com/BlueBubblesApp/BlueBubbles-Android-App/issues/786), [here](https://github.com/BlueBubblesApp/BlueBubbles-Android-App/issues/545), and [here](https://github.com/BlueBubblesApp/BlueBubbles-Android-App/issues/823)).

**Comment by CSab6482** [comment]
Here is a similar ticket for the Matrix iMessage bridge https://github.com/tulir/mautrix-imessage/issues/1

**Comment by ayancey** [comment]
Would love to +1 this and add a bounty. I have an old jailbroken iPhone that can stay plugged in 24/7

**Comment by CSab6482** [comment]
Good idea with the bounty, I'm willing to put up $150 USD for this feature.

**Comment by ayancey** [comment]
I’m not sure if there’s a legit way to do it but I will pledge $100 😂
